hypha/settings/base.py

```diff
@@ -20,7 +20,8 @@
 # Hypha custom settings
 
 # Set the currency symbol to be used.
-CURRENCY_SYMBOL = env.str('CURRENCY_SYMBOL', '$')
+CURRENCY_CODE = env.str('CURRENCY_CODE', 'USD')
+LOCALE = env.str('LOCALE', 'en_US')
```

Comment:
I would recommend picking this `locale` value from the `LANGUAGE_CODE` setting.

Reply:
That is logical but have some practical issues.

`LANGUAGE_CODE` specifies what locales files to read translations from. So for no one has translated Hypha into another language as far as I know. What if they still want to pick another currency?